test/core/call_indirect.wast

```diff
@@ -80,6 +80,38 @@
     ))
   )
 
+  (func
```

Comment:
Are we still trying to keep stacky/flat stuff in stack.wast (e.g. for Binaryen)? If not, I'd rather keep it here.

Reply:
Good point, moved to stack.wast